src/styles/features/pathways/app-bar.css

```diff
@@ -1,5 +1,7 @@
 .app-bar {
-  max-width: 80vw;
+  width: intrinsic;           /* Safari/WebKit uses a non-standard name */
+  width: -moz-max-content;    /* Firefox/Gecko */
```

Comment:
Generally should not use browser specific css, we pipe all source css to postCSS which should handle this.  Also it seems like intrinsic is experimental; It might not work in older versions of browsers.

Reply:
Shouldn't it suffice to use only `max-width` for specifying width?  I think it's more a matter of cutting off the name if it overflows the width or making the text wrap to the next line.  For wrapping, we would need to consider a max height as well.